neuronunit/optimization/evaluate_as_module.py

```diff
@@ -242,20 +192,97 @@ def pre_format(dtc):
     return dtc
 
 
+def cache_sim_runs(dtc):
```

Comment:
@russelljjarvis 
Is this a way of caching entire models (or rather their DTC versions) rather than caching results?  I want to confirm, so I don't later think this is a duplication of the general backend caching implementation.

Reply:
It was a duplicate.